src/python/hangman/README.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Lærerveiledning - Hangman
+level: 3
+language: nb
+---
+
+
+# Om oppgaven {.activity}
+
+I denne oppgaven skal vi lage det klassiske spillet Hangman. Der brukeren prøver å gjette seg frem til et hemmelig ord. 
+
+
+## Oppgaven passer til: {.check}
+
+ __Fag__: Programmering
```

Comment:
Satt leenge å prøvde å finne kompetansemål som kunne passe. Men problemet er at det akkuratt ikke går. F.eks passer nesten "lage tekster som kombinerer ord, lyd og bilde, med og uten digitale verktøy". Problemet her er at en ikke kombinerer bilde og lyd, selv om førstnevnte ville vært naturlig om en faktisk laget en ASCII strekmann. Er litt for grunnleggende til at jeg finner noe som passer.